lib/registries/addon/overview/template.hbs

```diff
@@ -96,6 +96,27 @@
                             {{/each}}
                         </div>
                     {{/if}}
+
+                    {{#let (unique-id 'resource-link') as |id|}}
+                        <leftNav.link data-analytics-name='Resources'
+                            @route='registries.overview.resources'
+                            @models={{array this.registration.id}}
+                            @icon='archive'
+                            @label={{t 'registries.overview.resources.title'}}
+                            id={{id}}
+                        />
+                        <NewFeaturePopover
+                            @triggerElement={{concat '#' id}}
+                            @featureCookie={{this.outputFeaturePopoverCookie}}
+                        >
+                            <:header>
+                                {{t 'registries.overview.files.outputFeaturePopoverHeading'}}
+                            </:header>
+                            <:body>
+                                {{t 'registries.overview.files.outputFeaturePopoverBody'}}
+                            </:body>
+                        </NewFeaturePopover>
+                    {{/let}}
```

Comment:
This used to be wrapped in an `#if` block. That's going to need to come back (and you'll probably want to wrap your stuff in the same if block to make sure that we don't show resources information when it's not possible to add resources to the registration).

Reply:
`{{#if this.registration.resourcesVisible}}` is the if statement that's missing around that block.

Reply:
Ok, this is re-added. Thank you